### ISSUE TITLE
Receiving `Error: Cannot read property 'toString' of undefined`

### ISSUE THREAD

**Issue by beaucharman** [jacobrask/eslint-plugin-sorting#18]
I am receiving the following error on some otherwise valid JSX / JavaScript files upon opening them when this plugin is active:

```
Error: Cannot read property 'toString' of undefined
--------------
TypeError: Cannot read property 'toString' of undefined
    at getKey (/node_modules/eslint-plugin-sorting/lib/rules/sort-object-props.js:12:25)
    at /node_modules/eslint-plugin-sorting/lib/rules/sort-object-props.js:29:35
    at Array.reduce (native)
    at EventEmitter.ObjectExpression (/node_modules/eslint-plugin-sorting/lib/rules/sort-object-props.js:20:33)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
    at NodeEventGenerator.enterNode (/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:607:23)
    at CommentEventGenerator.enterNode (/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
    at Controller.traverser.traverse.enter (/node_modules/eslint/lib/eslint.js:905:36)
```
I am fairly sure is has to do with dynamically created object members, for example when working with CSS Modules:

```
    {
      [styles.active]: isActive,
    }
```

Any idea what might be causing this? Thank you in advance for looking at this.

**Comment by jacobrask** [comment]
Thanks for your report, this should be fixed in master and will be released as soon as #17 is merged.

**Comment by jacobrask** [comment]
Can you confirm that the issue is fixed?

**Comment by laggingreflex** [comment]
This fixes it I hope. It specifically fixes error resulting from `{[styles.active]: isActive,}` type of properties. Please reopen if you still have issue. Published 0.2.2
